web/src/components/software/PatternSelector.jsx

```diff
@@ -166,6 +166,12 @@ function PatternSelector({ patterns, onSelectionChanged = noop }) {
     );
   });
 
+  if (selector.length === 0) {
+    selector = (
+      <Label>{_("No results")}</Label>
```

Comment:
I'would not use the `<Label>` component for this kind of things. Not even the Patternfly guys [would use it either](https://www.patternfly.org/components/label/design-guidelines#usage) :wink:

Not saying we have to follow strictly their design guidelines, but in this case I'm more aligned with them.

Use just plain old [HTML](https://html.spec.whatwg.org/multipage/) like [`<span>`](https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-span-element), [`<b>`](https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-b-element) or even [`<strong>`](https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-strong-element) if we considered it is something should be emphasise.

Also, do not hesitate of adding more text to make it clear. I cannot help too much with texts, but "No matches" looks a bit poor to me.